CharacterPanelRefinedPlugin.cs

```diff
@@ -310,25 +340,25 @@ public class CharacterPanelRefinedPlugin : IDalamudPlugin {
             offensivePtr->Y = 150;
             var dh = offensivePtr->ChildNode;
             dh->Y = 120;
-            dhChancePtr = AddStatRow((AtkComponentNode*)dh, "Direct Hit Chance");
+            dhChancePtr = AddStatRow((AtkComponentNode*)dh, Languages.Direct_Hit_Chance);
```

Comment:
Might need to add a char count test here to prevent accidentally adding localization strings that overflow the textbox.

Reply:
As the strings this would apply to are fairly limited, manually checking if they fit should be alright for now.

Reply:
Yeah, not something to be implemented for now, just a thought